### DISCUSSION TITLE
Build and publish your first GitHub Action

### DISCUSSION THREAD

**Discussion by giscus** [notiz-dev/notiz#231] [General]
# Build and publish your first GitHub Action

Build your first GitHub Action and deploy it to the Marketplace - Using TypeScript 💙

https://notiz.dev/blog/build-and-publish-your-first-github-action

**Comment by mnrendra** [comment]
Hi @garygrossgarten thank you for sharing. After I hitted the 'Release' button, I can't see the 'Publish this Action to the GitHub Marketplace' checkbox. Can you help me why it's happened? Thank you @garygrossgarten !

**Reply by garygrossgarten** [reply]
Hey @mnrendra,

Can you make sure to include all the necessary info in action.yml and also provide a readme in your repo.

![publish.png](https://user-images.githubusercontent.com/8986373/236991113-ccbafa5c-ea73-4895-8809-9e1012407909.png)

If you did all that maybe something has changed since i published the post.
